rca/static_src/javascript/components/carousel-full-peek.js

```diff
@@ -9,6 +9,7 @@ class PeekFullCarousel {
     constructor(node) {
         this.node = node;
         this.windowWidth = window.innerWidth;
+        this.tab = document.querySelectorAll('[data-tab]');
```

Comment:
as it's `querySelectorAll` might be worth naming the var to `this.allTabs`

Reply:
updated, thanks